src/Microsoft.DotNet.VersionTools/Automation/PullRequestCreator.cs

```diff
@@ -167,24 +167,54 @@ public class PullRequestCreator
                 $" * [`{oldCommit.Sha.Substring(0, 7)}`]({oldCommitsUrl}) {oldCommit.Message}\r\n" +
                 $"{statusLines}";
 
+            // Find comment to update, if one exists.
+            GitHubComment[] prIssueComments = await client.GetAllIssueCommentsAsync(
+                baseProject,
+                pullRequestToUpdate.Number);
+
+            GitHubComment commentToUpdate = prIssueComments
+                .FirstOrDefault(c => c.User.Login == _auth.User);
+
+            string commentBodyToUpdate = commentToUpdate?.Body ?? string.Empty;
```

Comment:
Why not just create a new comment each time? Is there any value in updating an existing comment?